script/DeployArbitrum.s.sol

```diff
@@ -14,41 +14,44 @@ contract DeployArbitrum is
     DeployBalancerV2FlashLoanCallback,
     DeployRadiantV2FlashLoanCallback
 {
-    address public constant DEPLOYER = 0xDdbe07CB6D77e81802C55bB381546c0DA51163dd;
+    address public constant DEPLOYER = 0xBcb909975715DC8fDe643EE44b89e3FD6A35A259;
+    address public constant OWNER = 0x64585922a9703d9EdE7d353a6522eb2970f75066;
+    address public constant PAUSER = 0x660Cc6D82925Cc804aC4EBD1d5870Fa32C9aBDb8;
+    address public constant DEFAULT_COLLECTOR = 0x3EBe4dfaF95cd320BF34633B3BDf773FbE732E63;
 
     /// @notice Set up deploy parameters and deploy contracts whose `deployedAddress` equals `UNDEPLOYED`.
     function setUp() external {
         create3FactoryConfig = Create3FactoryConfig({
-            deployedAddress: 0xB9504E656866cCB985Aa3f1Af7b8B886f8485Df6,
+            deployedAddress: 0xFa3e9a110E6975ec868E9ed72ac6034eE4255B64,
             deployer: DEPLOYER
```

Comment:
Since we have production and testing create3 factories ready for future deployment, I propose we remove the create3 factory deployment from all Deploy*.s.sol files except for DeployLocal.s.sol.

Reply:
I think we still need it considering new chain integrations and new create3 factory deployments in the future. Since we don't need to do anything right now, we can resolve this comment.